source/plot/svg/Shapes.ooc

```diff
@@ -1,31 +1,45 @@
 use ooc-math
+use ooc-draw
 
 Shapes: abstract class {
-	line: static func (start, end: FloatPoint2D, lineWidth: Float, opacity: Float, color: String) -> String {
-		"<line x1='" << start x toString() >> "' y1='" & start y toString() >> "' x2='" & end x toString() >> "' y2='" & end y toString() >> "' stroke-width='" & lineWidth toString() >> "' stroke='" >> color >> "' stroke-opacity='" & opacity toString() >> "'/>\n"
+	getColor: static func (bgra: ColorBgra) -> String {
+		"rgb(" clone() & bgra bgr red toString() & "," clone() & bgra bgr green toString() & "," clone() & bgra bgr blue toString() & ")" clone()
```

Comment:
This was written before we had the `>>` and `<<` operators. `&` frees the operators on both sides (unlike `+`, which frees neither), so in order for the literal strings here not to be freed, they had to be cloned. Trust me, it was even worse when we only had `+` - every intermediary `String` created had to be freed on its own line. 

This line can now be written as
```ooc
"rgb(" << bgra bgr red toString() >> "," & bgra bgr green toString() >> "," & bgra bgr blue toString() >> ")"
```
and that's the reason we now have the `Text` class, which we should use instead of changing the code to what I just wrote.